### PR TITLE
Explain head pose estimation adas 0001 coordinate system

### PR DESCRIPTION
**Pull request by Wovchena**
Close https://github.com/openvinotoolkit/open_model_zoo/issues/3558

### CODE REVIEW

**Comment by Wovchena** [comment]
@balegajay, please, take a look

File: models/intel/head-pose-estimation-adas-0001/README.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ Head pose estimation network based on simple, handmade CNN architecture. Angle r
 layers are convolutions + ReLU + batch norm + fully connected with
 one output.
 
+The estimator outputs yaw pitch and roll angels measured in degrees. Suppose the following coordinate system:
```
Comment (vladimir-dudnik) [inline models/intel/head-pose-estimation-adas-0001/README.md:9]:
```suggestion
The estimator outputs yaw pitch and roll angles measured in degrees. Suppose the following coordinate system:
```

Comment (Wovchena) [reply models/intel/head-pose-estimation-adas-0001/README.md]: Thank you!

**Comment by eaidova** [comment]
@Wovchena  @a-a-egorovich can we merge these changes?

**Review by eaidova — APPROVED** [review]

**Pull request merged**